src/modules/mavlink/mavlink_mission.cpp

```diff
@@ -272,7 +272,7 @@ MavlinkMissionManager::send_mission_current(uint16_t seq)
 
 		mavlink_msg_mission_current_send_struct(_mavlink->get_channel(), &wpc);
 
-	} else if (seq == 0 && item_count == 0) {
+	} else if (seq <= 0 && item_count == 0) {
 		/* don't broadcast if no WPs */
 
 	} else {
```

Comment:
since `seq` is int32_t now, on the next line it should be `%d` instead of `%u`:
`PX4_DEBUG("WPM: Send MISSION_CURRENT ERROR: seq %u out of bounds", seq);`